lib/ex_sanity/portable_text/serializers.ex

```diff
@@ -90,4 +92,15 @@ defmodule ExSanity.PortableText.Serializers do
       raise "a custom serializer for type: #{type} could not be found"
     end
   end
+
+  def render_custom_mark(serializers, mark, content) do
+    mark_serializers = serializers[:marks]
+    type_as_atom = String.to_atom(mark)
+
+    if Map.has_key?(mark_serializers, type_as_atom) do
+      mark_serializers[type_as_atom].(serializers, mark, content)
```

Comment:
Also, should this be `mark_serializers[type_as_atom].(serializers, content, mark)`? Are these arguments being called in the right order?

Reply:
I believe these are called in the correct order